meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -123,6 +123,13 @@ else
 fi
 inform "resin-init-flasher configuration found."
 
+# Find path to image
+BALENA_IMAGE=$(find / -xdev -type f -name "${BALENA_IMAGE}")
```

Comment:
Maybe do not override `BALENA_IMAGE` immediately? If it fails to find the file in `/`, it will always expand to empty string in the error message below.

Reply:
Thanks, I addressed that.